Containerfile.task

```diff
@@ -17,6 +17,18 @@ WORKDIR /app
 COPY dockerfile-json .
 RUN go build -o dockerfile-json
 
+FROM registry.access.redhat.com/ubi10/ubi-minimal@sha256:5a57b4c2509df8df587e19cc7c2d9cfa45b012139f5decd77f942daeb2334228 AS kubectl-builder
+
+ARG BUILDER_RPMS="golang git make"
+RUN microdnf install -y $BUILDER_RPMS
+
+ENV GOPROXY='https://proxy.golang.org,direct'
+ENV GOSUMDB='sum.golang.org'
+
+WORKDIR /app
+COPY kubectl .
+RUN make kubectl
```

Comment:
MO we can merge this into the previous step to save time (don't have to run `dnf install` twice, can reuse Go module cache for both builds)